### ISSUE TITLE
Show losses on the power flow diagram?

### ISSUE THREAD

**Issue by PhilPJL** [springfall2008/batpred#4789]
<img width="355" height="262" alt="Image" src="https://github.com/user-attachments/assets/b91e7911-742a-4c50-8888-89ca6b3f5025" />

The above diagram shows 4017 W  battery + 213 W solar, then 3330 export + 463 consumption.  This leaves a shortfall of 437 W.  Should the power flows reconcile to 0?  If it's losses can the diagram be updated to show them, without making things too cluttered I guess.

**Comment by springfall2008** [comment]
Automated first-pass triage — a maintainer will review before any action is taken.

**Classification:** enhancement (no priority assigned per triage guidelines for feature requests)

**Investigation:** The Power Flow diagram (`get_power_flow_diagram()` in `apps/predbat/web.py:585-865`) plots four independently-measured live values — `pv_power`, `battery_power`, `grid_power`, `load_power` (web.py:598-611) — which are each read directly from separate inverter sensor fields (`apps/predbat/inverter.py:1438-1468`, summed across inverters in `apps/predbat/execute.py:993-996`). None of the four is derived from the others, so nothing in the code enforces or expects them to balance to zero — the diagram has no reconciliation or loss calculation at all today.

This is the same underlying phenomenon reported in #2983 ("Loads not matching battery power"), where a maintainer confirmed the mismatch is normal — real DC→AC inverter conversion loss between the battery/PV side and the house/grid side, plus the four channels not necessarily being sampled at exactly the same instant.

Using your numbers (battery discharge 4017 W + PV 213 W generating = 4230 W in; export 3330 W + consumption 463 W = 3793 W out), the residual works out to:

```
loss = (pv_power + max(battery_power, 0) + max(-grid_power, 0))   # sources: PV, battery discharge, grid import
     - (house_power + max(-battery_power, 0) + max(grid_power, 0))  # sinks: house, battery charge, grid export
```

= 437 W here (~10% of the 4230 W sourced), which is plausible as inverter/battery conversion loss (the planner's own `inverter_loss`/`battery_loss` config default to 3-4% each, so a combined ~8-10% under a 4 kW discharge + PV pass-through isn't unreasonable) though some of it may also be timing skew between the independently-polled channels rather than pure conversion loss.

**Suggestion for implementation:** the residual above could be computed and shown as a small text label (e.g. "Loss: 437 W") rather than a full extra node/arrow, keeping the diagram from getting cluttered as you suggested. Left for maintainer judgement on whether/how to add this.

Related but distinct open requests against the same diagram: #4442 (routing), #4007 (car as separate node) — not duplicates of this ask.

No missing information needed — this doesn't require a log/debug yaml to investigate, since it's a design question about the diagram's math rather than a specific-environment bug.
